testing/wslu/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Patrick Wu <wotingwu@live.com>
+# Contributor: Patrick Wu <wotingwu@live.com>
+pkgname=wslu
+pkgver=2.3.2
+pkgrel=0
+pkgdesc="A collection of utilities for Windows 10 Linux Subsystems"
+url="https://github.com/wslutilities/wslu"
+arch="all"
+license="GPL-3.0-or-later"
+depends="bc bash-completion imagemagick"
+source="wslu-$pkgver.tar.gz::https://github.com/wslutilities/wslu/archive/v$pkgver.tar.gz"
```

Comment:
I think we usually keep `$pkgname-$pkgver.tar.gz` since it always makes sense to have the tarball called like the package. Sorry for the mess around this, we really need this in the official docs :)